pkg/wfs200/capabilities.go

```diff
@@ -123,12 +123,12 @@ type FeatureTypeList struct {
 
 // FeatureType struct for the WFS 2.0.0
 type FeatureType struct {
-	Name          string           `xml:"Name" yaml:"name"`
-	Title         string           `xml:"Title" yaml:"title"`
-	Abstract      string           `xml:"Abstract" yaml:"abstract"`
-	Keywords      *wsc110.Keywords `xml:"ows:Keywords" yaml:"keywords"`
-	DefaultCRS    *wsc110.CRS      `xml:"DefaultCRS" yaml:"defaultcrs"`
-	OtherCRS      *[]wsc110.CRS    `xml:"OtherCRS" yaml:"othercrs"`
+	Name          string             `xml:"Name" yaml:"name"`
+	Title         string             `xml:"Title" yaml:"title"`
+	Abstract      string             `xml:"Abstract" yaml:"abstract"`
+	Keywords      *[]wsc110.Keywords `xml:"ows:Keywords" yaml:"keywords"`
```

Comment:
Dit zorgt ervoor dat je keywords soms als array (met `- keywords:`) en soms als map (dus zonder `-`: `  keywords`) moet invullen. Ik ben hier niet super enthousiast over. Tegelijkertijd veranderd dit de yaml alleen daar waar het nodig is.

Reply:
dat was me ook al opgevallen, maar had genoeg andere issues om te tackelen :) Lijkt me prima, mits dit consistent is doorgevoerd in de applicatie.

Reply:
Ja dit moet ik dus gaan aanpassen in de operators. Will do.